src/main.rs

```diff
@@ -100,6 +144,18 @@ fn host_websocket(
                     }
                     info!("Video {}: host disconnected", video_id);
                 }).await;
```

Comment:
Problem: this iteration never stops, because it's on the channel not the websocket. Need to stop forwarding when the websocket is closed.